java.lang.IllegalMonitorStateException
	at java.lang.Object.wait(Native Method)
	at org.nlogo.job.JobThread.$anonfun$run$2(JobThread.scala:77)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.control.Exception$Catch.apply(Exception.scala:224)
	at org.nlogo.api.Exceptions$.ignoring(Exceptions.scala:38)
	at org.nlogo.job.JobThread.$anonfun$run$1(JobThread.scala:77)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.control.Exception$Catch.apply(Exception.scala:224)
	at org.nlogo.api.Exceptions$.handling(Exceptions.scala:41)
	at org.nlogo.job.JobThread.run(JobThread.scala:66)
Error in checkForRemoteErrors(val) : 
  one node produced an error: java.lang.Exception: Found unknown datatype: null
Calls: nl_run ... clusterApply -> staticClusterApply -> checkForRemoteErrors
Execution halted
srun: error: cg1-18: task 0: Exited with exit code 1
